### PR TITLE
Add setup instructions to README

### PR DESCRIPTION
**Pull request by arku**
I am not sure if we should add any other instructions. Let me know if I missed something.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ RTNS (RTrade Name Service) is a stand-alone IPNS record management service, desi
 
 It is essentially a modified and condensed version of [go-ipfs/namesys](https://github.com/ipfs/go-ipfs/tree/master/namesys) with minor optimizations
 
+# Setup
```
Comment (bonedaddy) [inline README.md]: This should be renamed to `Development`.

Other than looks good and is a much needed addition to the readme! :muscle: :rocket:

**Review by bonedaddy — CHANGES_REQUESTED** [review]
1 small change to be made and it's good to go :rocket: 

As for other instructions, I'm open to any suggestions but this should be good for now.

File: README.md (hunk 1 above)
Comment (arku) [reply README.md]: Done.

**Comment by arku** [comment]
@postables Can you PTAL? 🙂

**Comment by bonedaddy** [comment]
Looks good thanks :fire:

**Pull request merged**